theme

```diff
@@ -62,6 +62,7 @@
   "malcolmturnbull/draculaish-ha-theme",
   "Matt-PMCT/Green-and-Dark-HA-Theme",
   "mikosoft83/hass-windows10-themes",
+  "mjs271/arcticForest-dark_HA",
```

Comment:
Error:  <Validation hacsjson> failed:  The repository has an invalid 'hacs.json' file (More info: https://hacs.xyz/docs/publish/include#check-hacs-manifest )